wordpress/containerbuddy/memcached.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+consul-template \
+    -once \
+    -dedup \
+    -consul consul:8500 \
```

Comment:
We're getting smarter about injecting the Consul URL as a env var. We should do that in this image as well. See examples in:

- https://github.com/tgross/triton-elasticsearch/blob/master/etc/containerbuddy.json#L2
- https://github.com/tgross/triton-elasticsearch/blob/master/bin/manage.sh#L5